src/main/java/nwoolcan/view/MainController.java

```diff
@@ -0,0 +1,54 @@
+package nwoolcan.view;
+
+import javafx.application.Platform;
+import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.scene.Parent;
+import javafx.scene.layout.Pane;
+
+/**
+ * Handles the Main view.
+ */
+@SuppressWarnings("NullAway") // TODO
```

Comment:
Why is there a suppress warning and why is there a TODO? One line, two triggers.

Reply:
Since JavaFX creates controls using the FXML decorator, I must declare them but not initialize them (as you can see on line 16); if I do this NullAway gets angry. So instead of putting `@Nullable` for every single field I suppressed the warnings.

I put TODO because I don't know if it's the best way to handle this situation.